### PR TITLE
Release notes for 1.0.0-rc.1

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Release notes for 1.0.0-rc1

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: End game

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [X] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: docs/release_notes/v1.0.0-rc.1.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+If you previously installed Dapr using Helm, you can upgrade Dapr to a new version.
+If you installed Dapr using the CLI, go [here](#starting-fresh-install-on-a-cluster).
+
+##### 1. Get the latest CLI
+
+Get the latest version of the Dapr CLI as outlined above, and put it in your path.
+You can also use the helper scripts outlined [here](https://github.com/dapr/cli#installing-dapr-cli) to get the latest version.
+
+##### 2. Upgrade existing cluster
```
Comment (youngbupark) [inline docs/release_notes/v1.0.0-rc.1.md:261]: This part needs to be updated to this instruction - https://github.com/dapr/docs/pull/907/files

File: docs/release_notes/v1.0.0-rc.1.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+```bash
+dapr init
+```
+
+Wait for the update to finish,  ensure you are using the latest version of Dapr(1.0.0) with:
+
+```bash
+$ dapr --version
+
+CLI version: 1.0.0
```
Comment (mukundansundar) [inline docs/release_notes/v1.0.0-rc.1.md:245]: change to RC version.

File: docs/release_notes/v1.0.0-rc.1.md [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+
+  NAME                   NAMESPACE    HEALTHY  STATUS   REPLICAS  VERSION     AGE  CREATED
+  dapr-dashboard         dapr-system  True     Running  1         0.3.0       15s  2020-11-11 13:07.39
+  dapr-sidecar-injector  dapr-system  True     Running  1         1.0.0-rc1   15s  2020-11-11 13:07.39
+  dapr-sentry            dapr-system  True     Running  1         1.0.0-rc1   15s  2020-11-11 13:07.39
+  dapr-operator          dapr-system  True     Running  1         1.0.0-rc1   15s  2020-11-11 13:07.39
+  dapr-placement         dapr-system  True     Running  1         1.0.0-rc1   15s  2020-11-11 13:07.39
+```
+
+After Dapr 1.0.0 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
```
Comment (mukundansundar) [inline docs/release_notes/v1.0.0-rc.1.md:367]: change to RC

File: docs/release_notes/v1.0.0-rc.1.md [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+##### 2. Upgrade existing cluster
+
+Upgrading from 0.11.x to 1.0.0-rc.1 safely requires additional step to keep 0.11.x placement service to migrate from 0.11.x placement service to 1.0.0-rc.1 placement service. These two commands disable `helm upgrade` to uninstall 0.11.x placement service.
+
+```sh
+kubectl annotate deployment dapr-placement "helm.sh/resource-policy"=keep -n dapr-system
+kubectl annotate svc dapr-placement "helm.sh/resource-policy"=keep -n dapr-system
+```
+
+Then [export certs manually](#exporting-certs-manually).
```
Comment (youngbupark) [inline docs/release_notes/v1.0.0-rc.1.md:270]: please remove the link.

File: docs/release_notes/v1.0.0-rc.1.md [hunk 5]
```diff
@@ -0,0 +1,400 @@
...
+Update pods that are running Dapr to pick up the new version of the Dapr runtime.
+```sh
+kubectl rollout restart deploy/<DEPLOYMENT-NAME>
+```
+
+Once the deployment is completed, delete 0.11.x dapr-placement service by following commands:
+```sh
+kubectl delete deployment dapr-placement -n dapr-system
+kubectl delete svc dapr-placement -n dapr-system
+
```
Comment (youngbupark) [inline docs/release_notes/v1.0.0-rc.1.md:314]: \``` missing

File: docs/release_notes/v1.0.0-rc.1.md (hunk 1 above)
Comment (artursouza) [reply docs/release_notes/v1.0.0-rc.1.md:261]: Done.

File: docs/release_notes/v1.0.0-rc.1.md [hunk 6]
```diff
@@ -0,0 +1,401 @@
...
+```bash
+dapr uninstall --all
+```
+
+For RC releases like this, download the latest and greatest release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+Once you have installed the CLI, run:
+
+```bash
+dapr init
```
Comment (msfussell) [inline docs/release_notes/v1.0.0-rc.1.md:237]: As per our discussion this will be **dapr init --version=1.0.0-rc.1** otherwise even the 1.0.0-rc.1 will pull down the 0.11.3 release

**Review by youngbupark — APPROVED** [review]

File: docs/release_notes/v1.0.0-rc.1.md [hunk 7]
```diff
@@ -0,0 +1,397 @@
...
+
+We would like to extend our thanks to all the new and existing contributors who helped make this release happen.
+
+**Highlights**
+
+This is a release candidate version as we progress towards Dapr v1.0.0. We appreciate you trying Dapr and providing feedback on this release.
+
+> Dapr CLI version 1.0.0-rc.1 is not compatible with previous runtime versions. Download Dapr CLI 1.0.0-rc.1 from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+> For Java applications, use SDK version 0.10.0-rc-1.
```
Comment (yaron2) [inline docs/release_notes/v1.0.0-rc.1.md:14]: I would rephrase so that it doesn't seem like you must have an SDK.

Something like `For the Java SDK, use version 0.10.0-rc-1`.

Same thing for .Net below.

File: docs/release_notes/v1.0.0-rc.1.md [hunk 8]
```diff
@@ -0,0 +1,397 @@
...
+If you previously installed Dapr using Helm, you can upgrade Dapr to a new version.
+If you installed Dapr using the CLI, go [here](#starting-fresh-install-on-a-cluster).
+
+##### 1. Get the latest CLI
+
+Download the latest RC release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+##### 2. Upgrade existing cluster
+
+Upgrading from 0.11.x to 1.0.0-rc.1 safely requires additional step to keep 0.11.x placement service to migrate from 0.11.x placement service to 1.0.0-rc.1 placement service. These two commands disable `helm upgrade` to uninstall 0.11.x placement service.
```
Comment (yaron2) [inline docs/release_notes/v1.0.0-rc.1.md:266]: `These two commands disable `helm upgrade` to uninstall 0.11.x placement service.`

I don't understand what this sentence means, can we rephrase?

File: docs/release_notes/v1.0.0-rc.1.md [hunk 9]
```diff
@@ -0,0 +1,397 @@
...
+
+```sh
+kubectl annotate deployment dapr-placement "helm.sh/resource-policy"=keep -n dapr-system
+kubectl annotate svc dapr-placement "helm.sh/resource-policy"=keep -n dapr-system
+```
+
+Then export certs manually.
```
Comment (yaron2) [inline docs/release_notes/v1.0.0-rc.1.md:273]: Change to `Exports certs:`

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: docs/release_notes/v1.0.0-rc.1.md (hunk 8 above)
Comment (youngbupark) [reply docs/release_notes/v1.0.0-rc.1.md:266]: > Upgrading from 0.11.x to 1.0.0-rc.1 safely requires additional step to keep 0.11.x placement service to migrate from 0.11.x placement service to 1.0.0-rc.1 placement service. These two commands disable `helm upgrade` to uninstall 0.11.x placement service.

changed to 

> To upgrade from 0.11.x to 1.0.0, run the below commands to migrate from 0.11.x placement service to 1.0.0 placement service:

File: docs/release_notes/v1.0.0-rc.1.md [hunk 10]
```diff
@@ -0,0 +1,399 @@
...
+dapr-sentry-84565c747b-ldnsf             1/1     Running   0          29s
+dapr-sidecar-injector-68f868668f-6xnbt   1/1     Running   0          41s
+dapr-sidecar-injector-68f868668f-j7jcq   1/1     Running   0          29s
+dapr-sidecar-injector-68f868668f-ltxq4   1/1     Running   0          36s
+```
+
+
+Update pods that are running Dapr to pick up the new version of the Dapr runtime.
+```sh
+kubectl rollout restart deploy/<DEPLOYMENT-NAME>
```
Comment (msfussell) [inline docs/release_notes/v1.0.0-rc.1.md:312]: What is an example of the DEPLOYMENT-NAME? When was this set from the steps above?

Comment (artursouza) [reply docs/release_notes/v1.0.0-rc.1.md:312]: This depends on customer's application.

File: docs/release_notes/v1.0.0-rc.1.md (hunk 7 above)
Comment (artursouza) [reply docs/release_notes/v1.0.0-rc.1.md:14]: It is very unlikely for users not to use SDK to use Dapr in these 2 languages. I will rephrase it.

File: docs/release_notes/v1.0.0-rc.1.md (hunk 10 above)
Comment (artursouza) [reply docs/release_notes/v1.0.0-rc.1.md:312]: Reworded it.

**Review by yaron2 — APPROVED** [review]
lgtm
